02-introduction.rst

```diff
@@ -10,7 +10,8 @@ and your new friends (among others) are named "vectors", "arrays", "views" or
 Simple example
 --------------
 
-Let's take a very simple example, random walk.
+Let's take a very simple example, random walk. (You can execute any code below
+from the `code` folder, ideally inside an IPython session or Jupyter notebook.)
```

Comment:
I'm note sure I get your point about the `timeit` call. In fact, I rewrote the timeit function just to have the same (more or less) output as in Jupyter without having to use Jupyter. You mean experienced user will think this come from Jupyter ? If so, might be worth adding some explanation.